scripts/remove-quant-ph/script.py

```diff
@@ -0,0 +1,43 @@
+from inspirehep.curation.search_check_do import SearchCheckDo
+from inspire_utils.record import get_value
+import datetime
+
+
+DATE_BEFORE_2023 = datetime.datetime.strptime("2023", "%Y")
+
+
+class RemoveQuantPh(SearchCheckDo):
+    """Remove core from quant-ph literature records"""
+
+    query = "arxiv_eprints.categories:quant-ph core:true not _desy_bookkeeping.status:final  not _desy_bookkeeping.status:printed and de < 2023"
+
+    @staticmethod
+    def check(record, logger, state):
+        arxiv_category_quant_ph = (
+            get_value(record, "arxiv_eprints.categories") == "quant-ph"
```

Comment:
the left-hand side is a list of lists (as both `arxiv_eprints` and `categories` are arrays).